### ISSUE TITLE
changing HDMI monitor after installing berryboot

### ISSUE THREAD

**Issue by thisisnottaken** [maxnet/berryboot#415]
hi
I am using berryboot on raspberry pi 3.
I formatted my 16gb sd card as fat32 and copied the contents of berryboot-20170527-pi2-pi3.zip to it .
Now I booted my pi , then a setup screen that appeared that asked questions pertaining to audio and video setup. I answered . On subsequent reboots , I see boot menu . All good .
Now I changed my hdmi monitor switched from low res 1366x768 to  monitor having 1920x1080p.
Now screen appears cropped at borders.
How can I tell berryboot to give user that audio video setting survey again ?

Thanks.

**Comment by maxnet** [comment]
Press the "edit config" button in the Berryboot menu editor, go to `config.txt` tab and add:

```
disable_overscan=1
```

**Comment by thisisnottaken** [comment]
did not make any difference .
moreover , i am looking for this    https://ibb.co/jPFPDQ  screen to reappear . 
is that possible ?

**Comment by maxnet** [comment]
No, that's only possible if you haven't installed any OSes yet.
All that screen does for the video part is add disable_overscan=1 though...

**Comment by macmpi** [comment]
BTW, as noted here https://github.com/maxnet/berryboot/issues/411#issuecomment-309681365, overscan detection is failing on recent berryboot due to Pi model identification bug
